types/coord.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+)
+
+// Coord code for coordinates used throughout the API.
+// This is the Go representation of "Coord". It implements Place.
+// See http://doc.navitia.io/#standard-objects.
+type Coord struct {
+	Longitude float64 `json:"lon"`
+	Latitude  float64 `json:"lat"`
+}
+
+// jsonCoord define the JSON implementation of Coord struct
+type jsonCoord struct {
+	Latitude  string `json:"lat"`
```

Comment:
eventhough it's not a time, its good to have units in the name of the variable, or at least in a comment (see https://github.com/uber-go/guide/blob/master/style.md#use-timetime-and-timeduration-with-external-systems)

Reply:
I'm agree, I'm gonna change it and fields name to match perfectly to the API. Here's a preview:

```go
type Coord struct {
	Lon float64 `json:"lon"` // Longitude in degrees
	Lat  float64 `json:"lat"`  // Latitude in degrees
}
```

Reply:
Changes has been reverted to avoid breaking changes